lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -62,14 +63,20 @@ def self.run(params)
           end
         end
 
-        fad_api_client.update_release_notes(release_name, release_notes(params))
+        release_notes = release_notes(params)
+        if release_notes.nil? || release_notes.empty?
+          UI.success("✅ No release notes passed in. Skipping this step.")
```

Comment:
nit: different icon for "skip"?  🚫? ⏩? ⬜? 🗨️ ?

Reply:
Good call! Went with ⏩